components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -27,6 +27,30 @@
     <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+    <alias>CMPASO-JRA1p5</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_DICE%SSMI_MPASO%DATMFORCED_DROF%JRA-1p4-2018-AIS0LIQ_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
+
+  <compset>
+    <alias>CMPASO-JRA1p5-PISMF</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_DICE%SSMI_MPASO%PISMFDATMFORCED_DROF%JRA-1p4-2018-AIS0LIQ_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
+
+  <compset>
+    <alias>CMPASO-JRA1p5-DISMF</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_DICE%SSMI_MPASO%DISMFDATMFORCED_DROF%JRA-1p4-2018-AIS0LIQ_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
+
+  <compset>
+    <alias>CMPASO-JRA1p5-PISMF-TMIX</alias>
+    <lname>2000_DATM%JRA-1p5_SLND_DICE%SSMI_MPASO%PISMFDATMFORCEDTMIX_DROF%JRA-1p4-2018-AIS0LIQ_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
```

Comment:
@irenavankova, I think we want to take these out of this PR.  If we want to add new C-case compsets, we probably want that in its own small PR.

Reply:
Removed